blog/chris_F_pydata2022.md

```diff
@@ -0,0 +1,189 @@
+
+## Introduction
+
+Bayesian statistical methods offer a powerful set of tools to tackle a wide variety of data science problems. In addition, the Bayesian approach generates results that are easy to interpret and automatically account for uncertainty in quantities that we wish to estimate and predict. Historically, computational challenges have been a barrier, particularly to new users, but there now exists a mature set of probabilistic programming tools that are both capable and easy to learn. We will use the newest release of PyMC (version 4) in this tutorial, but the concepts and approaches that will be taught are portable to any probabilistic programming framework.
+
+This tutorial is intended for practicing and aspiring data scientists and analysts looking to learn how to apply Bayesian statistics and probabilistic programming to their work. It will provide learners with a high-level understanding of Bayesian statistical methods and their potential for use in a variety of applications. They will also gain hands-on experience with applying these methods using PyMC, specifically including the specification, fitting and checking of models applied to a couple of real-world datasets.
+
+As this is an introductory tutorial, no direct experience with PyMC or Bayesian statistics will be required. However, to benefit maximally from the tutorial, learners should have some familiarity with basic statistics (things like regression and estimation) and with core components of the scientific Python stack (e.g. NumPy, pandas and Jupyter).
+
+As the goal of the tutorial is to get new users up and running with Bayesian methods, the content is light on theory and focus on the implementation of models, though some statistical background will be provided for context and clarity. Since PyMC is a high-level statistical package, it is easy to gloss over important details of the underlying algorihtms. Therefore, the tutorial begins by solving a simple model using only NumPy and SciPy functions before diving into PyMC. As a capstone to the tutorial, learners will be introduced to "The Bayesian Workflow" to reiterate the important steps in the process, along with useful tips and tricks.
+
+## About Speaker
+
+Chris is the Principal Quantitative Analyst in Baseball Research & Development for the Philadelphia Phillies. He is interested in computational statistics, machine learning, Bayesian methods, and applied decision analysis. He hails from Vancouver, Canada and received his Ph.D. from the University of Georgia.
+
+- GitHub: https://github.com/fonnesbeck
+- LinkedIn: https://www.linkedin.com/in/christopher-fonnesbeck-374a492a
+
+
+## Video
+
+<br>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/911d4A1U0BE?start=3" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<br>
+
+## Outline
+
```

Comment:
The outline was what was provided before the actual talk so combining it with the talk might be not so ideal, it might be repetitive or not in place. The timestamps are more beneficial since they highlight the exact flow of the talk and I think even if we add them as sections to the timestamps it wont make any difference and maybe make the timestamps less specific.

Alternatively, we can do away with the outline  and remain with only the timestamps since they are more important.

Let me know what you think.

Reply:
removing the outline sounds good then